[master < E-mage-export] Add MAGE export import documentation (#1035)
memgraph/docs: mage/query-modules/python/import-util.md

<RunOnSubgraph/>

### `cypher(path)`

> Imports the cypher queries from the given path by just running the queries.

Reviewer: ```suggestion
Imports the Cypher queries from the given path by just running the queries.
```

#### Input:

> * `path: string` ➡ Path to the Cypher file that is being imported.

Reviewer: ```suggestion
* `path: string` ➡ path to the Cypher file that needs to be imported.
```
